projects/ad40xx_fmc/zed/system_bd.tcl

```diff
@@ -1,4 +1,5 @@
 
+source $ad_hdl_dir/library/spi_engine/scripts/spi_engine.tcl
```

Comment:
We should do the process call inside the common block design script. Being common for all carriers.